src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,149 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
```

Comment:
We should include our max-width includes here.

Reply:
Yep I agree